web/locales/cs.json

```diff
@@ -662,7 +662,7 @@
    "raw_transaction_details" : "Raw Transaction Details",
    "recent_blocks" : "Recent Blocks",
    "recent_transactions" : "Poslední transakce",
-   "recipient" : "Příjemce",
+   "recipientAddress" : "Příjemce",
```

Comment:
recipient_address would be more consistent

Reply:
Agreed, unfortunately this is how it comes back from the service. This is why some cell values are translated and others are not. What we need is some sort of parser, a camelCaseToUnderlineLowercase function would probably do the trick for 90% of the cell values. I'l see about that in a follow-up PR.